kmp-library/src/commonMain/kotlin/uk/co/lucasweb/aws/v4/signer/hash/ByteExtensions.kt

```diff
@@ -0,0 +1,12 @@
+package uk.co.lucasweb.aws.v4.signer.hash
+
+/**
+ * Convert a Byte into an unsigned Int.
+ * Source: https://stackoverflow.com/questions/38651192/how-to-correctly-handle-byte-values-greater-than-127-in-kotlin
+ *
+ * TODO: Remove this once Kotlin 1.3 is released as it brings support for UInt
```

Comment:
Isn't Kotlin 1.3 already released? https://blog.jetbrains.com/kotlin/2019/01/kotlin-1-3-20-released/

Reply:
It is released, but changing the type to [UInt](https://kotlinlang.org/api/latest/jvm/stdlib/kotlin/-u-int/index.html) would require us changing the SHA-256 implementation, as these types `Int` vs. `UInt` are not compatible. Ideally, I'd like to leave this to the authors of the hashing implementation.